react/react2/week1/github_user/src/Components/APIOutput.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useContext, useEffect, useState } from "react";
+import UserList from "./UserList";
+import Blocker from "./Blocker";
+import { SearchInputContext } from "./Contexts";
+export default function APIOutput() {
+  const [data, setData] = useState();
```

Comment:
We use useState to define the variables with initial value to use them locally. defining useState () with null or empty string or empty array or empty object or Boolean etc is a good practice.